modules/holiday-snow.php

```diff
@@ -108,6 +108,19 @@ function jetpack_holiday_snow_option_name() {
 	return apply_filters( 'jetpack_holiday_snow_option_name', 'jetpack_holiday_snow_enabled' );
 }
 
+function jetpack_show_holiday_snow_option() {
+	// Always show snow option if a custom snow season has been set.
+	if ( has_filter( 'jetpack_is_holiday_snow_season' ) ) {
+		return true;
+	}
+
+	$today            = time();
+	$first_option_day = mktime( 0, 0, 0, 11, 24 ); // Nov 24
+	$last_option_day  = mktime( 0, 0, 0, 1, 11 );  // Jan 11
```

Comment:
Ok, I can change the end date to match the holiday period easily enough. I integrated @georgestephanis' "[bike shedding](https://github.com/Automattic/jetpack/issues/6065#issuecomment-271661925)" 😛